src/Kunstmaan/AdminListBundle/Event/AdminListEvent.php

```diff
@@ -12,12 +14,26 @@ class AdminListEvent extends Event
     protected $entity;
 
     /**
+     * @var FormInterface
+     */
+    protected $form;
+
+    /**
+     * @var Request
+     */
+    protected $request;
+
+    /**
      * AdminListEvent constructor.
      * @param object $entity
+     * @param FormInterface $form
+     * @param Request $request
      */
-    public function __construct($entity)
+    public function __construct($entity, Request $request, FormInterface $form = null)
```

Comment:
Personally, I do not like `nullable` if it is not necessary :) it depends on case. We can allow BC I guess, as we do not have stable version for v4

@jockri